### PR TITLE
add rockcraft.yaml for kubeflow-profiles

### PR DESCRIPTION
**Pull request by ca-scribner**

### CODE REVIEW

File: profile-controller/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  profile-controller:
+    override: replace
+    command: /manager
+    startup: enabled
+
+parts:
+  profile-controller:
+    plugin: go
+    source-type: git
+    source: https://github.com/kubeflow/kubeflow.git
```
Comment (i-chvets) [inline profile-controller/rockcraft.yaml:27]: Need to specify branch or a tag, eg. `source-tag: v1.7.0-branch`

File: profile-controller/rockcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+build-base: ubuntu:22.04
+
+license: Apache-2.0
+
+platforms:
+  amd64:
+
+services:
+  profile-controller:
+    override: replace
```
Comment (i-chvets) [inline profile-controller/rockcraft.yaml:19]: Add summary for service

File: profile-controller/rockcraft.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      go build -a -o $CRAFT_PART_INSTALL/manager main.go
+
+      mkdir -p $CRAFT_PART_INSTALL/third_party/library
+      cp -pr $HOME/go/pkg/mod/github.com/hashicorp $CRAFT_PART_INSTALL/third_party/library
+    stage:
+      - manager
+      - third_party/library/hashicorp
+    prime:
+      - manager
+      - third_party/library/hashicorp
```
Comment (i-chvets) [inline profile-controller/rockcraft.yaml:51]: Add non-root part

Comment (i-chvets) [inline profile-controller/rockcraft.yaml:51]: Add security-team-requirement part.

File: profile-controller/rockcraft.yaml (hunk 1 above)
Comment (ca-scribner) [reply profile-controller/rockcraft.yaml:27]: Added

File: profile-controller/rockcraft.yaml (hunk 2 above)
Comment (ca-scribner) [reply profile-controller/rockcraft.yaml:19]: added

File: profile-controller/rockcraft.yaml (hunk 3 above)
Comment (ca-scribner) [reply profile-controller/rockcraft.yaml:51]: This is for the thing you're asking about in the rockcraft channel?  What should I add here?  We know the current code doesn't quite work, but should I add it anyway to keep the same as everyone else?

Comment (ca-scribner) [reply profile-controller/rockcraft.yaml:51]: Added

**Review by i-chvets — APPROVED** [review]
Approved.

**Review by i-chvets — APPROVED** [review]
Approved.

**Pull request merged**